docs/source/author/writing_guide.rst

```diff
@@ -435,6 +393,17 @@ Video (Verto feature)
 
 ------------------------------------------------------------------------------
 
+Writing structure
+==============================================================================
+
+Chapters always contain the following:
+  
+  - An introduction page and several chapter sections that each cover a key sub topic of the main chapter topic.
+  - End in the same two sections; 'The whole story!' and 'Further reading'.
+    Where 'The whole story!' is a summary of the chapter and 'Further reading' contains links and/or book references where students can read further about the topic.
+
```

Comment:
Consider saying this:
```
Chapters always contain the following:

  - An introduction page, which introduces the reader to the chapter.
  - Several chapter sections, which each cover a key sub topic of the main chapter.
  - A ``The whole story!`` page, which should mention some other related concepts not covered by the chapter.
  - A ``Further reading`` page, which contains links and/or book references where students can investigate the topic further.
```

Note that I described 'The whole story!' completely differently. That's my interpretation from reading a few of the namesakes from existing chapters

Also the config files for chapters talk about a video associated with the chapter. That of course refers to the intro video at the very beginning of each chapter, but I don't know if one is *required*

Reply:
Video isn't required :) for example the introduction to coding doesn't have a video url listed in it's config file.

Reply:
Thanks for the rewritten content!